src/writing-your-own-application/nostd.md

```diff
@@ -1,17 +1,17 @@
-# Writing no_std applications
+# Writing `no_std` Applications
 If you want to learn how to develop `no_std` application, see the following training materials:
-- The book [Embedded Rust (no_std) on Espressif][no-std-book]
-- The repository [esp-rs/no_std-training][no-std-repository]
+- The book [Embedded Rust (`no_std`) on Espressif][no-std-book]
+- The repository [`no_std-training`][no-std-repository]
 
 The training is based on [ESP32-C3-DevKit-RUST-1][esp-rust-board]. You can use any other Espressif development board but code changes and configuration changes might be needed.
 
 The training contains:
 * Introductory level examples:
-   * A basic hello-world ([Source][hello-world])
-   * A panic example ([Source][panic])
-   * A blinky example ([Source][blinky])
-   * A button example ([Source][button])
-   * A button with interrupt example ([Source][button-interrupt])
+   * [A basic hello-world][hello-world]
```

Comment:
I like that you removed "Source" here, because the text "a ... example" makes very good link text itself.